config/betterquesting/resources/supersymmetry/lang/en_us.lang

```diff
@@ -55,17 +55,17 @@ susy.quest.ql.26.desc=There are a few creatures you may encounter.
 susy.quest.ql.27.title=Power Generation
 susy.quest.ql.27.desc=Let's make those electrons §odance§o.§r For progress.
 susy.quest.db.0.title=Genesis
-susy.quest.db.0.desc=Welcome to §9§9§9§b§b§b§b§b§b§l§l§l§l§l§lSupersymmetry§r§r§r§r§r§r! %n%nYou have just landed on an unfamiliar planet. This book will teach you how to rebuild civilization from scratch in only §k4200§r pages.%n%nStart by getting a rock§r. This quest book is interactive.%n%nWhen you pick up a rock from the ground, press the §l§l§l§l§l§l§a§a§a§a§a§aSUBMIT§r button to complete the quest and continue.%n%nIf you lose this book, you can check the quests by pressing the §2§aGRAVE (`) §rkey.%n%nPages that have a normal icon aren't required for progression. %nPages that have a framed icon, like this one are §crequired §rand contain critical information for your survival.%n%nYou have been given a §6home block§r. It allows you to set your spawn if you ever die. You cannot skip the night, as beds are not available before EV.%n%n
+susy.quest.db.0.desc=Welcome to §9§9§9§b§b§b§b§b§b§l§l§l§l§l§lSupersymmetry§r§r§r§r§r§r! %n%nYou have just landed on an unfamiliar planet. This book will teach you how to rebuild civilization from scratch in only §k4200§r pages.%n%nStart by getting a rock§r. This quest book is interactive.%n%nWhen you pick up a rock from the ground, press the §l§l§l§l§l§l§a§a§a§a§a§aSUBMIT§r button to complete the quest and continue.%n%nIf you lose this book, you can check the quests by pressing the §2§aGRAVE (`) §rkey.%n%nPages that have a normal icon aren't required for progression. %nPages that have a framed icon (like this one are!) §crequired §rand contain critical information for your survival.%n%nYou have been given a §6home block§r. It allows you to set your spawn if you ever die. You cannot skip the night, as beds are not available before EV.%n%n
```

Comment:
Should be:
(like this one!) are
(Do not put "are" in the parentheses)

Reply:
fixed